Fix some doc issues (#111)
epinio/docs: docs/howtos/install_wordpress_application.md

Wordpress needs a database to work. After visiting the route of your deployed
application you will have to set the connection details to the database.

You can use the [Service](../references/services.md) feature to deploy your database with Epinio.

> Of course, deploying a database by your own is still possible.

Reviewer: ```suggestion
Of course, deploying a database manually is still possible.
```
